pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/block/base/Beacon.kt

```diff
@@ -0,0 +1,13 @@
+package io.github.pylonmc.pylon.core.block.base
+
+import com.destroystokyo.paper.event.block.BeaconEffectEvent
+import io.papermc.paper.event.block.BeaconActivatedEvent
+import io.papermc.paper.event.block.BeaconDeactivatedEvent
+import io.papermc.paper.event.player.PlayerChangeBeaconEffectEvent
+
+interface Beacon {
+    fun onActivated(event: BeaconActivatedEvent)
```

Comment:
```suggestion
    fun onActivated(event: BeaconActivatedEvent) {}
```
These should all be default empty so that we won't have to do empty impls

Reply:
at least find and replace exists...